src/pytest_ansible_units/__init__.py

```diff
@@ -0,0 +1,177 @@
+"""Setup the collection for testing."""
+from __future__ import annotations
+
+import logging
+import os
+import sys
+from pathlib import Path
+
+import pytest
+from _pytest.config.argparsing import Parser
+
+logger = logging.getLogger(__name__)
+
+try:
+    import yaml
+
+    HAS_YAML = True
+except ImportError:
+    HAS_YAML = False
+
+try:
+    HAS_ANSIBLE = True
+except ImportError:
+    HAS_ANSIBLE = False
+
+try:
+    from ansible.utils.collection_loader._collection_finder import (
+        _AnsibleCollectionFinder,
+    )
+
+    HAS_COLLECTION_FINDER = True
+except ImportError:
+    HAS_COLLECTION_FINDER = False
+
+
+def pytest_addoption(parser: Parser) -> None:
+    """Add the options to the pytest command.
+
+    :param parser: The pytest parser object
+    """
+    parser.addoption(
+        "--inject-only",
+        action="store_true",
+        default=False,
+        help="Only inject the current ANSIBLE_COLLECTIONS_PATHS",
+    )
+
+
+def pytest_configure(config: pytest.Config) -> None:
```

Comment:
This `pytest_configure` code can be moved, if necessary, into the pytest_ansible/plugin file